packages/api-plugin-fulfillment/src/schemas/schema.graphql

```diff
@@ -0,0 +1,414 @@
+  rate: Float!
+}
+
+"Provider info"
+type ProviderInfo{
+  "Name of the provider"
+  name: String!
+  
+  "Label of the provider"
+  label: String!
+
+  "Flag defining enabled/disabled status"
+  enabled: Boolean
+}
+"Fulfillment type root object"
+type FulfillmentTypeObj implements Node {
+  "The Fulfillment Type ID"
+  _id: ID!
+
+  "The user provided name of the fulfillment type"
+  name: String!
+
+  "Shop ID"
+  shopId: ID!
+
+  "ProviderInfo"
+  provider: ProviderInfo
+
+  "Fulfillment type"
+  fulfillmentType: String
+
+  "Fulfillment methods"
+  methods: [FulfillmentMethodObj]
+}
+
+"Custom Fulfillment method object "
+type CustomFulfillmentMethodObj implements Node {
+  "The Fulfillment Type ID"
+  fulfillmentTypeId: ID!
+
+  "Shop ID"
+  shopId: ID!
+
+  "The fulfillment method ID"
+  _id: ID!
+
+  "The cost of this fulfillment method to the shop, if you track this"
+  cost: Float
+
+  """
+  The fulfillment types for which this method may be used. For example, `shipping` or `digital`.
+  """
+  fulfillmentTypes: [FulfillmentType]
+
+  "The group to which this method belongs"
+  group: String!
+
+  "A fixed price to charge for handling costs when this fulfillment method is selected for an order"
+  handling: Float!
+
+  "Include this as a fulfillment option shown to shoppers during checkout?"
+  enabled: Boolean!
+
+  "The name of this method, for display in the user interface"
+  label: String!
+
+  "The name of this method, a unique identifier"
+  name: String!
+
+  "The common-name for this method, to group all variants of the same method"
+  fulfillmentMethod: String
+
+  "A fixed price to charge for fulfillment costs when this fulfillment method is selected for an order"
+  rate: Float!
+}
+
+"Input needed to select a fulfillment option for a single fulfillment group on a cart"
+input updateFulfillmentTypeInputInfo {
+  "The shop to which this group belongs"
+  shopId: ID!
+
+  "The group to update"
+  fulfillmentGroupId: ID!
+
+  "The type name to be updated"
+  name: String!
+
+  "The label to be updated"
+  label: String!
+
+  "The displayMessage to be updated"
+  displayMessageType: String
+
+  "Flag to define if the group should be enabled/disabled"
+  enabled: Boolean
+}
+"Input needed to select a fulfillment option for a single fulfillment group on a cart"
+input updateFulfillmentTypeInput {
+  "The shop to which this group belongs"
+  groupInfo: updateFulfillmentTypeInputInfo!
+
+  "An optional string identifying the mutation call, which will be returned in the response payload"
+  clientMutationId: String
+}
+"The updated group info from updateFulfillmentType"
+type updateFulfillmentTypeGroup {
+  "The shop to which this group belongs"
+  shopId: ID!
+
+  "The group which was updated"
+  fulfillmentGroupId: ID!
+
+  "The updated group name"
+  name: String!
+
+  "The updated group label"
+  label: String!
+
+  "Flag defining enabled/disabled status"
+  enabled: Boolean
```

Comment:
Make this field required. (Refer https://github.com/reactioncommerce/reaction/pull/6570#discussion_r1004039840)

Reply:
@zenweasel 
**Update**: 
I had made this field as 'required' as per our earlier discussion. But now, I have reverted it back to 'optional'. Reason being, this is part of EDIT/UPDATE input and we are making all editable fields optional. 
Meanwhile, in the create input we have this field as 'required'.